### PR TITLE
Help command

### PR DESCRIPTION
**Pull request by jiaweiteo**

### CODE REVIEW

File: src/test/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/edit/EditActivityCommandTest.java [hunk 1]
```diff
@@ -78,8 +78,10 @@ public void execute_filteredList_success() {
 
         String expectedMessage = String.format(MESSAGE_EDIT_ACTIVITY_SUCCESS, editedActivity);
 
+
```
Comment (jeannetoh99) [inline src/test/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/edit/EditActivityCommandTest.java:81]: Is the additional line here intentional?

**Review by jeannetoh99 — APPROVED** [review]
Commented on a minor issue, but otherwise LGTM.

File: src/main/resources/view/TravelPlannerPanel.fxml [hunk 2]
```diff
@@ -8,7 +8,7 @@
 <?import javafx.scene.control.ListCell?>
 <VBox fx:id="travelPlanPanel" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
     <padding>
-        <Insets top="5" right="5" bottom="5" left="5" />
+        <Insets top="5" right="5" bottom="5" left="5"  />
```
Comment (lyeyixian) [inline src/main/resources/view/TravelPlannerPanel.fxml:11]: Why the whitespace!!!

**Review by lyeyixian — APPROVED** [review]
LGTM

**Pull request merged**